tasks/aiida-rest.yml

```diff
@@ -16,13 +16,30 @@
     daemon_user: "{{ current_user }}"
     daemon_user_home: "{{ current_user_home }}"
     daemon_aiida_venv: "{{ aiida_venv |  regex_replace('\\$\\{HOME}', current_user_home) }}"
+  when: ansible_service_mgr == "systemd"
 
-- name: Start AiiDA REST API system service
+- name: Start AiiDA REST API service
   become: true
   become_user: "{{ root_user }}"
   systemd:
     name: aiida-rest@{{ aiida_profile_name }}
     enabled: "{{ aiida_rest_service_enabled }}"
     masked: false
     daemon-reload: true
-  when: not (cloud_platform is defined and cloud_platform == 'docker')
+  when: ansible_service_mgr == "systemd"
+
+- block:
+    - name: Check if the AiiDA REST API is running
+      shell: pgrep -a verdi | grep restapi | awk '{print $1}'
+      changed_when: false
+      register: aiida_rest_status
+      failed_when: aiida_rest_status.rc not in [0,1]
+
+    - name: Start AiiDA REST API
+      become: true
```

Comment:
again, the user can do this

Reply:
Indeed. Thanks for pointing this out. Fixed.